### DISCUSSION TITLE
Bosch 18V Charger

### DISCUSSION THREAD

**Discussion by matthewdominick** [aderusha/DDD-Printable-Wall-Control-System#21] [Q&A]
I was able to modify the Milwaukee 12V Charger stl to work with a [Bosch 18V Charger](https://www.amazon.com/Bosch-GXS18V-15N15-Starter-CORE18V-Compact/dp/B07QD8LT2D/ref=sr_1_6?crid=W7618IGU6E0B&keywords=GAL18V-40&qid=1640962060&s=hi&sprefix=gal18v-40%2Ctools%2C77&sr=1-6). I am finishing up a few more tweaks to the design (printing, testing, printing . . .). I was going to make a pull request but don't have permissions. Do you want the the file for your repo?

**Comment by aderusha** [comment]
Hey @matthewdominick - this sounds great and I would love to include your model when you're ready!  The way GH works is that you'll need to [fork the repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo), add your file to your fork, then [issue a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) back to the main project.  If you go through this process your GH user will be attached to the main project as a contributor.

If you'd rather not deal with all that, simply post the file somewhere that I can access it and I'll add it to the repo under my username.

**Reply by matthewdominick** [reply]
No problem.  I'll make a fork when the stl is complete that includes the stl and an update to the docs. Thank you!
